tests/cypress/app.R

```diff
@@ -0,0 +1,36 @@
+library(teal)
+
+data <- teal_data() |>
+  within({
+    set.seed(123)
+    size <- 20
+    data_frame <- data.frame(
+      id = seq(size),
+      numeric = round(rnorm(size, 25, 10)),
+      logical = sample(c(TRUE, FALSE), size = size, replace = TRUE),
+      factor = factor(sample(LETTERS[1:4], size = size, replace = TRUE)),
+      character = sample(letters, size = size, replace = TRUE),
+      datetime = sample(
+        seq(as.POSIXct("2000/01/01"), as.POSIXct("2024/01/01"), by = "day"),
+        size = size
+      )
+    )
+    iris <- head(iris, 20)
+    data_frame$date <- as.Date(data_frame$datetime)
+  })
+datanames(data) <- c("data_frame", "iris")
+app <- teal::init(
```

Comment:
It should be a generic shiny app instead of teal. Testing could be distorted by `teal`. I suggest to use shiny app from vignette. Should also test the output if and how data gets filtererd?